### ISSUE TITLE
[Information] Validation response-json from backend to frontend

### ISSUE THREAD

**Issue by solojuve1897** [j3k0/cordova-plugin-purchase#824]
When implementing your own validation-service, communicating with Apple and Google, the following is what needs to be in the response you send from the backend to the frontend:

```
{
 "ok" [boolean]: true,
 "data" [object]: {
    "code" [integer]: for iOS set it to "status".
    "transaction" [object]: for iOS I set it to the last item in "latest_receipt_info"-array*,
   "latest_receipt" [string]: for iOS set it to "latest_receipt"
  }
}
```

* Not completely sure about this one actually. The last item seems to always be the latest transaction.

** With regards to Android I don't know yet since I'm just finished with iOS. Will update this with Android-info when I'm done.

I think for when there is an error one should instead return the following, but I'm not completely sure:

```
{
  "ok": false,
  "data": {
    "code": 6778003 (for example),
    "error": {
      "message": "blah blah"
    }
  }
}
```

My backend verifies the receipts in iOS and everything seems to be ok according to the debug output. Next in line is Android.

Why this hasn't been documented yet is beyond my understanding. Instead one has to study the code to figure out what's going on. Who has the knowledge if the above is 100% correct? If it is, I'm ready to make a pull-request if needed to get it published in the documentation.

**Comment by albertleao** [comment]
Thank you for this!

I've been trying to implement it myself for unit testing at the very least.

**Comment by solojuve1897** [comment]
@albertleao 

Actually after I wrote this I started questioning if I had everything covered to get this right with Apples horrible subscription service. When googling a bit I stumbled upon this:

https://github.com/RevenueCat/cordova-plugin-purchases

RevenueCat has a free validation service for Google and Apple and a newly implemented Cordova-plugin. Deserves to be checked out. I have decided to use that from now on an drop this plugin and the building of a validation service all together.
